DashAI/front/.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
+
+[*]
+end_of_line = lf
```

Comment:
I hope this lets windows users use CRLF line end when building

Reply:
This file configures the text editor to set line breaks to lf by default. But it is only enabled by default in pycharm (and others). To enable it in vscode, you have to install _EditorConfig for VS Code_.